web/angular-wallet/src/app/main/messages/sidenavs/right/options/options.component.ts

```diff
@@ -15,9 +15,12 @@ import { ActivatedRoute } from '@angular/router';
 export class MessageOptionsSidenavComponent implements OnInit, OnDestroy
 {
     options: any;
+
     @Input('fees') fees: SuggestedFees;
+    @Input('encrypt') encrypt: boolean;
```

Comment:
this is a cool feature, although I can imagine that final users might complain, because it's almost hidden

Reply:
yeah I thought the same, but its also kinda nice and simple, which i like 🤷‍♂️